go/libkb/keyfamily.go

```diff
@@ -728,6 +739,36 @@ func (ckf ComputedKeyFamily) GetActivePGPKeys(sibkey bool) (ret []*PGPKeyBundle)
 	return
 }
 
+type RevokedKey struct {
+	Key       GenericKey
+	RevokedAt *KeybaseTime
+}
+
+func (ckf ComputedKeyFamily) GetRevokedKeys() []RevokedKey {
+	var revokedKeys []RevokedKey
+	for kid := range ckf.kf.AllKIDs {
+		ki := ckf.cki.Infos[kid]
+		if ki == nil {
+			continue
```

Comment:
My thought with this loop is that all it is doing is finding revoked keys.  Any errors in the key family would be handled elsewhere.  So you feel like GetRevokedKeys finding an error is the key family should cause an error in LoadUserPlusKeys?